pymc4/util/interceptors.py

```diff
@@ -12,7 +12,7 @@
 VariableDescription = collections.namedtuple('VariableDescription', 'Dist,shape,rv')
 
 
-class Interceptor(object):
+class Interceptor():
```

Comment:
good practice to inherit from object, even if you don't need to!

Reply:
Why? I thought in py3 we don't need that anymore.

Reply:
Yes, I am facing a linting error when I import from `object`